league/oauth2-server-bundle/0.11/config/routes/league_oauth2_server.yaml

```
@@ -0,0 +1,3 @@
+oauth2_server:
```

For consistency improvement:
```suggestion
league_oauth2_server:
```